src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -42,10 +39,7 @@ const useGroupInfo = () => {
   const [isAnonymousModalOpen, setIsAnonymousModalOpen] = React.useState(false);
   const [isFetchingAllowAnonDM, setIsFetchingAllowAnonDM] = React.useState(false);
   const [, dispatchChannel] = React.useContext(Context).channel;
-
-  const {goToChatScreen} = useChatUtilsHook();
-  const {localDb} = useLocalDatabaseHook();
-
+  const {createSignChat, handleAnonymousMessage} = useCreateChat();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new hook `useCreateChat` is being used here. Ensure that this hook is properly tested and handles all edge cases, as it seems to be a critical part of the chat creation process.



<!-- This is an auto-generated comment by OSS CodeRabbit -->